doc/api/cli.md

```diff
@@ -1110,6 +1110,14 @@ Path to the file used to store the persistent REPL history. The default path is
 `~/.node_repl_history`, which is overridden by this variable. Setting the value
 to an empty string (`''` or `' '`) disables persistent REPL history.
 
+### `NODE_REPL_EXTERNAL_MODULE=file`
```

Comment:
I imagine reviewers will say "Instead of an environment variable, make this a CLI flag so that `NODE_OPTIONS` can be used."

Reply:
It says `NODE_REPL_EXTERNAL_MODULE` here but `NODE_EXTERNAL_REPL_MODULE` everywhere else.